### PR TITLE
[apiConformance] Fix relative all

### PR DESCRIPTION
**Pull request by sbalandi**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp [hunk 1]
```diff
@@ -62,8 +62,8 @@ class APIBaseTest : public CommonTestUtils::TestsCommon {
 
     void SetUp() override {
         set_api_entity();
-        auto test_name = this->GetTestName();
-        k = test_name.find("_mandatory") != std::string::npos || test_name.find("mandatory_") != std::string::npos ? 1 : 0;
+        auto suite_name = this->GetSuiteName();
```
Comment (iefode) [inline src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp:65]: test_name should contain test_suite name

File: src/tests/ie_test_utils/functional_test_utils/src/summary/api_summary.cpp [hunk 2]
```diff
@@ -69,12 +69,13 @@ void ApiSummary::updateStat(ov_entity entity, const std::string& target_device,
     if (isCrashReported) {
         cur_stat[real_device].crashed--;
         isCrashReported = false;
+    } else {
```
Comment (iefode) [inline src/tests/ie_test_utils/functional_test_utils/src/summary/api_summary.cpp:72]: we need to report cur_stat[real_device].rel_all += rel_influence_coef; for all cases. take a look op _report

File: src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp (hunk 1 above)
Comment (sbalandi) [reply src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp:65]: no, for test 
TEST(IEClassBasicTest, smoke_registerNonExistingPluginFileThrows)

test_name will contain only smoke_registerNonExistingPluginFileThrows , suite_name will contains instigation prefix too

Comment (iefode) [reply src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp:65]: And what about test name? We can set prefix for test_name also..

File: src/tests/ie_test_utils/functional_test_utils/src/summary/api_summary.cpp (hunk 2 above)
Comment (iefode) [reply src/tests/ie_test_utils/functional_test_utils/src/summary/api_summary.cpp:72]: Agree :) forget my code :D

File: src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp (hunk 1 above)
Comment (sbalandi) [reply src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp]: yes, you are right ! updated

**Pull request merged**